src/components/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+const initialState = [];
+
+function addBook(book) {
+  return {
+    type: 'ADD_BOOK',
+    payload: book,
+  };
+}
+
+function removeBook(id) {
+  return {
+    type: 'REMOVE_BOOK',
+    payload: id,
+  };
+}
+
+export default function reducer(state = initialState, action) {
+  if (action.type === 'ADD_BOOK') {
+    return [...state, action.payload];
+  }
+
+  if (action.type === 'REMOVE_BOOK') {
+    return state.filter((book) => book.id !== action.payload);
+  }
+  return state;
+}
```

Comment:
- [ ] [Optional] [Highly recommended] Please use arrow function syntax for all functions instead of regular function syntax.